### ISSUE TITLE
setDlcClothes can cause massive lags

### ISSUE THREAD

**Issue by cmdflow** [altmp/altv-issues#1210]
### Description of the problem

Setting clothes with setDlcCloths can result in massiv lags

### Reproduction steps

// This is the clothing data stored in database, every line is one user. The clothes are read via getDlcClothes and stored in db
// after login the user gets his clothes via command below:
 
```
{"1":[0,0,0,0],"3":[0,0,0,0],"4":[6,10,2,0],"5":[0,0,0,0],"6":[1,5,2,0],"7":[0,0,0,0],"8":[0,5,2,0],"9":[0,0,0,0],"10":[0,0,0,0],"11":[0,3,2,0]},
{"1":[0,0,2,0],"3":[0,0,2,0],"4":[0,0,2,0],"5":[19,0,2,1267047138],"6":[1,1,2,0],"7":[0,5,2,920492965],"8":[15,0,2,0],"9":[0,0,2,0],"10":[0,2,2,920492965],"11":[1,0,2,0]},
{"1":[0,0,2,0],"3":[0,0,2,0],"4":[0,0,2,0],"5":[19,0,2,1267047138],"6":[1,1,2,0],"7":[0,5,2,920492965],"8":[15,0,2,0],"9":[0,0,2,0],"10":[0,2,2,920492965],"11":[1,0,2,0]},
{"1":[0,0,2,0],"3":[0,0,2,0],"4":[0,0,2,0],"5":[19,0,2,1267047138],"6":[1,1,2,0],"7":[0,5,2,920492965],"8":[15,0,2,0],"9":[0,0,2,0],"10":[0,2,2,920492965],"11":[1,0,2,0]},
{"1":[0,0,2,0],"3":[0,0,2,0],"4":[0,0,2,0],"5":[19,0,2,1267047138],"6":[1,1,2,0],"7":[0,5,2,920492965],"8":[15,0,2,0],"9":[0,0,2,0],"10":[0,2,2,920492965],"11":[1,0,2,0]}
```
 
// command to set clothes on player (executed when player spawns in)
```
async function seClothesSetAll(player, data) {
    if(!data || typeof data === 'string') return;
    for await(let i of Object.keys(data)) {
        player.setDlcClothes(parseInt(data[i][3]), parseInt(i), data[i][0], data[i][1], data[i][2]);
    }
}
```
 
// Tested the following. All players (6 in test) spawn in, no clothes is set. Player meet on same spot running in circles, setting the clothes on all players do cause extreme laggs
 
// Testcommand:
```
function setClothes() {
    for (let play of alt.Player.all) {
        alt.emit("seClothes:set:all", play, player.data.clothes); // is function seClothesSetAll above
    }
}

```
### Expected behaviour

setdlcclothes should not cause massive lags. maybe you can implement validations for clothes or such

### Additional context

was reportet in discord. posted it here in tracker. you can decide if its a bug or not :)

### Operating system

debian 9

### Version

4.6

**Comment by FabianTerhorst** [comment]
i will add texture, drawable checks for gta dlcs. And will make the setclothes methods return true or false.

**Comment by cmdflow** [comment]
The only thing wich makes me really wonder is, that this are standard clothes. in a very low range of cloths meaning they are one of the first clothes implemented ever to GTA. They are set via setclothes and read via getDlcClothes, stored in database and on char connect set with setDlcClothes. So getclothes has to be:
a) not working with setclothes
b) reading some infos wrong
I cleared all clothes and we set the very lowest IDs to test. I am pretty shure this is kind of bug. And the lag is really massive. Thanks for concidering :)
